hack/deploy-logging.sh

```diff
@@ -150,75 +236,67 @@ wait_for_logging_is_running() {
+    done
+    SUFFIX="-clo" \
+    CONFIGMAP_NAME=clo-olm \
+    CREATE_OPERATORGROUP=false \
+    NAMESPACE=${LOGGING_NS} \
+    VERSION=${MASTER_VERSION} \
+    OPERATOR_IMAGE=$cloimg \
+    MANIFEST_DIR=${manifest} \
+    TEST_NAMESPACE=${LOGGING_NS} \
+    TARGET_NAMESPACE=${LOGGING_NS} \
+    hack/vendor/olm-test-script/e2e-olm.sh
 }
 
 deploy_logging_using_clo_make() {
-    if [ ! -d $GOPATH/src/github.com/openshift/elasticsearch-operator ] ; then
-        git clone https://github.com/${EO_REPO:-openshift}/elasticsearch-operator \
-            $GOPATH/src/github.com/openshift/elasticsearch-operator -b ${EO_BRANCH:-master}
-    fi
-    if [ ! -d $GOPATH/src/github.com/openshift/cluster-logging-operator ] ; then
-        git clone https://github.com/${CLO_REPO:-openshift}/cluster-logging-operator \
-            $GOPATH/src/github.com/openshift/cluster-logging-operator -b ${CLO_BRANCH:-master}
-    fi
     # edit the deployment manifest - use the images provided by CI or from api.ci registry
     # make deploy-no-build
-    EO_IMAGE=$( construct_image_name elasticsearch-operator latest )
-    CLO_IMAGE=$( construct_image_name cluster-logging-operator latest )
-    pushd $GOPATH/src/github.com/openshift/cluster-logging-operator > /dev/null
-    REMOTE_CLUSTER=true REMOTE_REGISTRY=true NAMESPACE=openshift-logging \
+    EO_IMAGE=${EO_IMAGE:-$( construct_image_name elasticsearch-operator latest )}
+    CLO_IMAGE=${CLO_IMAGE:-$( construct_image_name cluster-logging-operator latest )}
+    pushd $CLO_DIR > /dev/null
+    cp manifests/${MASTER_VERSION}/cluster-logging.*.clusterserviceversion.yaml manifests/${MASTER_VERSION}/cluster-logging.*.clusterserviceversion.yaml.orig
+    update_images_in_clo_yaml manifests/${MASTER_VERSION}/cluster-logging.*.clusterserviceversion.yaml $cloimg
+    REMOTE_CLUSTER=true REMOTE_REGISTRY=true NAMESPACE=$LOGGING_NS \
         IMAGE_OVERRIDE="$CLO_IMAGE" EO_IMAGE_OVERRIDE="$EO_IMAGE" make deploy-no-build
+    mv manifests/${MASTER_VERSION}/cluster-logging.*.clusterserviceversion.yaml.orig manifests/${MASTER_VERSION}/cluster-logging.*.clusterserviceversion.yaml
     popd > /dev/null
 }
 
 DEFAULT_TIMEOUT=${DEFAULT_TIMEOUT:-600}
 
 switch_to_admin_user
 
-USE_OLM=${USE_OLM:-false}
-
-if [ -z "${USE_CUSTOM_IMAGES:-}" ] ; then
-    if [ -n "${OPENSHIFT_BUILD_NAMESPACE:-}" -a -n "${IMAGE_FORMAT:-}" ] ; then
-        USE_CUSTOM_IMAGES=true
-    elif [ "${USE_IMAGE_STREAM:-false}" = true ] ; then
-        USE_CUSTOM_IMAGES=true
-    elif [ "${USE_CLO_LATEST_IMAGE:-false}" = true -o "${USE_EO_LATEST_IMAGE:-false}" = true ] ; then
-        USE_CUSTOM_IMAGES=true
-    else
-        # default to false - false means "use whatever images are defined in the default deployment"
-        USE_CUSTOM_IMAGES=false
-    fi
-fi
+USE_OLM=${USE_OLM:-true}
 LOGGING_NS=${LOGGING_NS:-openshift-logging}
-ESO_NS=${ESO_NS:-openshift-operators-redhat}
+ESO_NS=${ESO_NS:-openshift-logging}
```

Comment:
Did we modify this here for simplicity?  Any production deployment will use something other then `openshift-logging` which will be `openshift-operators-redhat` AFAIK